packages/node_modules/@webex/plugin-meetings/src/meeting-info/index.js

```diff
@@ -95,7 +95,7 @@ export default class MeetingInfo extends StatelessWebexPlugin {
     return MeetingInfoUtil.generateOptions({
       destination,
       type,
-      conversationServiceUrl: this.webex.internal.device.services.conversationServiceUrl
+      webex: this.webex
```

Comment:
why pass the full webex now instead?

Reply:
Because now we can't read from that, we have to get the clusterId and verify that the convo url is the cluster being looked at.